projects/ci-cd.md

```diff
@@ -0,0 +1,85 @@
+| K8S            | [Keptn](https://lifecycle.keptn.sh/)                                                                                                                        | [Keptn GitHub Repo](https://github.com/keptn/lifecycle-toolkit)                                    | * Focused on observability in Kubernetes deployments                          |
+| Jenkins        | [OpenTelemetry plugin](https://plugins.jenkins.io/opentelemetry/)                                                                                           | [Plugin GitHub Repo](https://github.com/jenkinsci/opentelemetry-plugin)                            | * Tracing of Jenkins pipelines                                                |
+| Jenkins        | [OpenTelemetry Agent Host Metrics](https://plugins.jenkins.io/opentelemetry-agent-metrics/)                                                                 |                                                                                                    | * Metrics for Jenkins hosts                                                   |
+| CLI            | [OTEL CLI](https://github.com/equinix-labs/otel-cli)                                                                                                        |                                                                                                    | * Tracing through a CLI wrapper                                               |
+| GitHub         | [OpenTelemetry Export Trace - Inception health](https://github.com/marketplace/actions/opentelemetry-export-trace)                                          | [GitHub Action Repo](https://github.com/inception-health/otel-export-trace-action)                 | * Traces via a GitHub action                                                  |
+| GitHub         | [OpenTelemetry Actions Exporter - New Relic](https://github.com/marketplace/actions/new-relic-opentelemetry-github-actions-exporter)                        | [GitHub Action Repo](https://github.com/newrelic-experimental/gha-new-relic-exporter)              | * Metrics on workflow runs                                                    |
+| GitHub         | [Run with Telemetry GitHub Action](https://github.com/krzko/run-with-telemetry)                                                                             |                                                                                                    | * Traces via GitHub action                                                    |
+
+
+## Staffing / Help Wanted
+
+The goal of this project will consist of three stages:
+
+1. Working group preparation
+2. Specification
+3. Implementation
+
+### Required staffing
+
+All stages will require a project lead, domain experts, and engineers at a minimum. 
+
+* @horovits (tentative project lead)
+* @adrielp
+* @thisthat
```

Comment:
```suggestion
* @thisthat
* @xibz
```

As domain expert for CD, specifically Spinnaker, and CDEvent
Timezone: UTC-6

Reply:
Added! 🎉